plugins/iis_logs.yaml

```diff
@@ -0,0 +1,84 @@
+    default: "UTC"
+  - name: include_file_name
+    type: bool
+    default: true
+  - name: include_file_path
+    type: bool
+    default: true
+  - name: include_file_name_resolved
+    type: bool
+    default: false 
+  - name: include_file_path_resolved
+    type: bool
+    default: false
+  - name: start_at
+    type: string
+    supported:
+      - beginning
+      - end
+    default: end
+template: |
+  receivers: 
+    filelog:
+      include: 
+      {{ range $i, $fp := .file_path }}
+        - '{{ $fp }}'
+      {{ end }}
+      {{ if .exclude_file_log_path }}  
+      exclude: 
+      {{ range $i, $efp := .exclude_file_log_path }}
+        - '{{ $efp }}'
+      {{ end }}
+     {{ end }}
+      include_file_name: {{ .include_file_name }}
+      include_file_path: {{ .include_file_path }}
+      include_file_name_resolved: {{ .include_file_name_resolved }}
+      include_file_path_resolved: {{ .include_file_path_resolved }}
+      start_at: {{ .start_at }}
+      operators: 
+    
+      - id: add_log_type
+        type: add
+        field: 'attributes.log_type'
+        value: 'microsoft_iis'
+
+      - id: body_parser
+        type: regex_parser
+        regex: '^(?P<timestamp>\d{4}-\d{2}-\d{2}\s+\d{2}:\d{2}:\d{2})\s+(?P<address>[^\s]+)\s+(?P<operation>\w{3})\s+(?P<cs_uri_stem>[^\s]+)\s(?P<cs_uri_query>[^\s]+)\s+(?P<s_port>[^\s]+)\s+-\s+(?P<remoteIp>[^\s]+)\s+(?P<userAgent>[^\s]+)\s+-\s+(?P<status>\d{3})\s+(?P<sc_status>\d)\s+(?P<sc_win32_status>\d)\s+(?P<time_taken>[^\n]+)'
+        timestamp:
+          parse_from: attributes.timestamp
+          layout: '%Y-%m-%d %H:%M:%S'
+          location: {{.location}}
+        severity: 
+          parse_from: attributes.status
+          mapping:
+            info: 2xx
+            info2: 3xx
+            warn: 4xx
+            error: 5xx
+        output: end_filter
+
+      - id: end_filter
+        type: filter
+        expr: 'body == ""'
```

Comment:
Do you need this? I added it in other plugins that had a diverging pipeline in order to bring them back together. Do you expect some logs to have a blank body and want to filter those out?